peepingtom/gui/widgets/volume_selector.py

```diff
@@ -0,0 +1,58 @@
+from qtpy.QtWidgets import QVBoxLayout, QWidget, QComboBox, QLabel
+
+
+class VolumeSelector:
+    def __init__(self, viewer):
+        self.viewer = viewer
+        self._selected = None
+
+        self._init_gui()
+
+        if self.volumes:
+            self.select_from_dropdown()
+
+    def _init_gui(self):
+        self.widget = QWidget()
+        layout = QVBoxLayout()
+        self.widget.setLayout(layout)
+
+        layout.addWidget(QLabel('PeepingTom - Volume Selector'))
```

Comment:
'PeepingTom - Volume Selector' is used here and in the `add_dock_widget` call - refactor into `VolumeSelector.name`?

Reply:
Ok, did it, more or less!